Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -119,6 +119,11 @@ public final class SwiftLanguageServer: ToolchainLanguageServer {
 
   var commandsByFile: [DocumentURI: SwiftCompileCommand] = [:]
 
+  var documentLookaheadRanges: [DocumentURI: LookaheadRanges] = [:]
```

Comment:
I think we should store the `LookaheadRanges` in the `DocumentSnapshot` as I mentioned here: https://github.com/apple/sourcekit-lsp/pull/761#discussion_r1255417400

Or is there a reason why we can’t do that?

I noticed this just now again by trying to write a doc comment for it and I couldn’t come up with a very nice explanation for it.

Reply:
Ah, seems I mis-understood that comment.